mailpoet/assets/css/src/components-automation-editor/_panel.scss

```diff
@@ -20,3 +20,35 @@
     padding: 4px;
   }
 }
+
+.mailpoet-step-name-dropdown {
+  display: block;
+
+  h2 {
+    margin: 0;
+  }
+
+  .mailpoet-automation-panel-plain-body-title-text {
+    padding-left: 0;
+    padding-top: 0;
+  }
+
+  .mailpoet-automation-panel-plain-body-title-action {
+    margin-top: -13px;
+    padding-right: 0;
+    padding-top: 0;
+  }
+}
+
+.mailpoet-step-name-input,
+.mailpoet-step-name-description,
```

Comment:
I'm not seeing the step name description get the same styling that we have for the instructions in the automation renaming thing, such as the lighter grey text and slightly smaller font size.

## Figma
<img width="206" alt="image" src="https://user-images.githubusercontent.com/8656640/184235410-da7d9f67-1e52-4004-a70d-1d05e4d33181.png">

## My local
<img width="284" alt="image" src="https://user-images.githubusercontent.com/8656640/184235477-c457ab2c-c3c0-4d9e-ae62-5be4c4d850ce.png">

## My local automation renaming thing
<img width="283" alt="image" src="https://user-images.githubusercontent.com/8656640/184235541-baa2c252-8c97-4c09-90e8-a1a34b87563f.png">